eng/tools/generator/cmd/v2/automation/automationCmd.go

```diff
@@ -131,14 +131,14 @@ func (ctx *automationContext) generate(input *pipeline.GenerateInput) (*pipeline
 				errorBuilder.add(err)
 				continue
 			}
-
+			packageRelativePath := tsc.GetRelativePath()
```

Comment:
could you rename with adding a prefix: `module`?

Reply:
done